src/helpers/SettingsProvider.ts

```diff
@@ -35,6 +35,10 @@ export class SettingsProvider {
     );
   }
 
+  getBasePromptInstruction(): string {
+    return this._config.get(" ") as string;
```

Comment:
Is this meant to just be a space? I'd expect to see `return this._config.get("basePromptInstruction") as string;`

Reply:
No idea how that happened... thanks for catching that